HIP/hip-991.md

```diff
@@ -318,6 +328,257 @@ To comply with this HIP, the Mirror Node provides the following features:
   }
   ```
 
+* `/api/v1/transactions`
+  * Changes to the body of the response. The response should include the new `max_custom_fees` field to expose any data defined in its corresponding protobuffer definition. A sample response payload follows.
+
+  ```json
+  {
+    "transactions": [
+      {
+        "bytes": null,
+        "charged_tx_fee": 7,
+        "consensus_timestamp": "1234567890.000000007",
+        "entity_id": "0.0.2281979",
+        "max_custom_fees": [
+          {
+            "fees": [
+              {
+                "amount": 1000,
+                "denominating_token_id": null
+              },
+              {
+                "amount": 500,
+                "denominating_token_id": "0.0.2000"
+              }
+            ],
+            "payer_account_id": "0.0.8"
```

Comment:
At the cost of diverging slightly from the protobuf, we'd prefer to duplicate the account id into the max fees in the JSON response to make the structure flatter and simpler.

```suggestion
          {
            "account_id": "0.0.8"
            "amount": 1000,
            "denominating_token_id": null
          },
          {
            "account_id": "0.0.8"
            "amount": 500,
            "denominating_token_id": "0.0.2000"
          }
```